### ISSUE TITLE
[feature request] List releases by build ID or upload date

### ISSUE THREAD

**Issue by madiguzel** [microsoft/appcenter-cli#1027]
Hi,

"appcenter distribute releases list" command currently lists builds by "Short version". We need to list them by build IDs or upload dates. Here is a current sample output:

```
appcenter distribute releases list -a ${AppName} 
ID: 75 
Short Version: 8.3.0 
Version: 1933 
Uploaded At: Tue Jul 14 2020 12:20:24 GMT+0300 (GMT+03:00)
ID: 78 
Short Version: 8.2.1 
Version: 1955 
Uploaded At: Thu Aug 20 2020 10:56:02 GMT+0300 (GMT+03:00)
ID: 77 
Short Version: 8.2.1 
Version: 1954 
Uploaded At: Wed Aug 19 2020 15:27:14 GMT+0300 (GMT+03:00)
ID: 76 
Short Version: 8.2.1 
Version: 1952 
Uploaded At: Tue Aug 18 2020 12:14:25 GMT+0300 (GMT+03:00)
ID: 74 
Short Version: 8.2.0 
Version: 1930 
Uploaded At: Tue Jul 14 2020 10:49:30 GMT+0300 (GMT+03:00)
```

Thank you for support.

**Comment by NTDBANNU** [comment]
Hi @russelarms @jp-andre when can we expect this enhancement in place to list releases based on build ID?

**Comment by NTDBANNU** [comment]
Hi @madiguzel, Currently I'm facing the same issue. could you please share if you have any workaround to get the releases based on build ID or upload date?

**Comment by madiguzel** [comment]
Hi @NTDBANNU, I've used json output format and used `jq` to sort the builds by ID.

```
latest_release_id=`appcenter distribute releases list -a "$organization/$appname" --output json | jq 'max_by(.id) | .id'`
```

**Comment by unnati914** [comment]
could i work on this?

**Comment by AnastasiaKubova** [comment]
Hi, @unnati914 , feel free to open contribution PR with a solution to this issue and we will definitely consider it.

**Comment by DmitriyKirakosyan** [comment]
As we do not have plans to add support for this feature in the next year, I'm closing the issue.
